LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
**(style|non-blocking)**

I think we generally go with `LICENSE.md` https://github.com/uclahs-cds/template-R-package/blob/main/LICENSE.md